packages/render-fidelity-tools/test/config.json

```diff
@@ -224,14 +224,16 @@
       "name": "khronos-MetalRoughSpheresNoTextures",
       "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheresNoTextures/glTF/MetalRoughSpheresNoTextures.gltf",
       "orbit": {
-        "radius": 12.0
+        "radius": 0.0000000005
```

Comment:
This feels like it's asking for a rounding error. Can you increase this to something reasonable by changing target z? If you subtract the same amount from z that you add to radius it should keep the camera in the same place (although I might have a sign wrong).

Reply:
Sigh, I couldn't fully figure this out